docs/install.rst

```diff
@@ -12,7 +12,7 @@ care of with installation using pip:
 * `numpy <https://numpy.org/>`__ 1.17.0 or later
```

Comment:
Version numbers of various packages (including python) need to be updated here as well.

Reply:
Updated.